HAPNodeJSClient.js

```diff
@@ -399,11 +427,13 @@ HAPNodeJSClient.prototype.HAPeventByDeviceID = function(deviceID, body, callback
  *
  * @param  {type} ipAddress IP Address of homebridge instance
  * @param  {type} port      Port of homebridge instance
+ * @param  {type} pin       Pin of homebridge instance
  * @param  {type} body      An array of HomeKit characteristic updates, [{ \"aid\": 2, \"iid\": 9, \"value\": 0}]
  * @param  {type} callback  Callback to execute upon completion of characteristic setting, function(err, response)
  */
 
-HAPNodeJSClient.prototype.HAPevent = function(ipAddress, port, body, callback) {
+HAPNodeJSClient.prototype.HAPevent = function(ipAddress, port, pin, body, callback) {
```

Comment:
This is a breaking change to the external API and can not be merged as it will break all other packages using this package